packages/cli/internal/pkg/aws/cdk/progress.go

```diff
@@ -51,6 +51,18 @@ func (p ProgressStream) DisplayProgress(description string) error {
 	return nil
 }
 
+func SilentExecution(progressStreams []ProgressStream) []Result {
```

Comment:
We should follow DRY principles here. This function can be merged with `ShowExecution`. We can have the single function take a function as a second parameter that does the `if event.LastOutput != ""` check and apply that function in the for loop. For `SilentExecution`, that function would just be a 1->1 mapping, but for the `ShowExecution` version, the function can execute the log call.